sql/parser2/encode.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2012, Google Inc. All rights reserved.
```

Comment:
What's up with this copyright notice attributed to Google, vs. others in this PR which are for Cockroach?